config/jobs/kubernetes-csi/OWNERS

```diff
@@ -3,8 +3,10 @@
 reviewers:
 - msau42
 - saad-ali
+- xing-yang
```

Comment:
As followup, @jsafrane and @pohly should be added to approvers as well

Reply:
sure.